packages/dashboard/e2e/README.md

```diff
@@ -13,24 +43,6 @@ Below is a diagram representing the flow of commands when running `npm run test:
 
 ## Container and network interactions
 
-Below is a diagram of how the containers and networks interact when up and running
-
-![Containers and network interaction diagram](docs/resources/e2e-Container-networks.png)
-
-Broadly speaking, there are 3 major parts of the system:
-
-- Romidashboard container, built from the `docker.pkg.github.com/osrf/rmf-web/e2e` image with the RMF backend compiled into it. It holds the ROS2 bridge server and the React dashboard. Sourcing of RMF is required before running the tests and it can be sourced from `. /opt/rmf/setup.bash`
-- Auth container
-- auth_network, a known network where the Auth container is reachable 
-
-Requests from the romidashboard to the auth container would be routed to the gateway ip address of `auth_network`. Docker would then perform a NAT to route the requests to the auth container and vice versa.
+The key difference between running the tests locally and in github workflows is that in github, the tests are ran from inside a container with docker-beside-docker. So the docker commands connects to the host daemon. The main difference this cause is that "localhost" in github refers to the container, but "localhost" in local runs refers to the host (where the docker daemon is running), as such, the tests wouldn't be able to connect to the auth service because when you "expose" or "publish" a port, it is mapped to the host, not the container.
```

Comment:
nit suggestions 

I changed some wording:

So, the docker commands connect to the host daemon,  this causes that the "localhost" in GitHub refers to the container. "Localhost" in local runs refers to the host (where the docker daemon is running). Tests in local runs wouldn't be able to connect to the auth service because when a port is "exposed" o "published" is mapped to the host and not the container.

HTH,